python/ray/scripts/scripts.py

```diff
@@ -691,6 +691,7 @@ def stop(force, verbose, log_style, log_color):
         ["log_monitor.py", False],
         ["reporter.py", False],
         ["dashboard.py", False],
+        ["new_dashboard/agent.py", False],
```

Comment:
cc @edoakes incidental fix so ray stop kills the leaked agents

Reply:
Oh haha I just opened an issue about that.

Reply:
Oh man, we should really have a test for this...